chef/cookbooks/hyperv/recipes/openstack_install_nova.rb

```diff
@@ -24,3 +24,14 @@
   EOH
   not_if { ::File.exists?("#{node[:openstack][:nova][:installed]}") }
 end
+
+ruby_block "ensure correct python path in shebang" do
```

Comment:
this looks like code duplication
can this be moved to a helper and be used from both places?